web/src/main/java/com/marklogic/hub/web/web/FlowController.java

```diff
@@ -131,8 +131,8 @@
 
     @RequestMapping(value = "/{flowName}/run", method = RequestMethod.POST)
     @ResponseBody
-    public ResponseEntity<?> runFlow(@PathVariable String flowName, @RequestBody(required = false) List<String> steps) {
-        FlowStepModel flow = flowManagerService.runFlow(flowName, steps);
+    public ResponseEntity<?> runFlow(@PathVariable String flowName, @RequestBody(required = false) Object steps) {
+        FlowStepModel flow = flowManagerService.runFlow(flowName, new ArrayList<>());
```

Comment:
The UI bug has been fixed. We don't require this, can revert to old code.

Reply:
OK